compute/cloud-client/src/test/java/compute/SnippetsIT.java

```diff
@@ -118,6 +131,25 @@ public static void cleanup() throws IOException, InterruptedException, Execution
     System.setOut(null);
   }
 
+  public static String getBase64EncodedKey() {
+    String sampleSpace = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+    StringBuilder stringBuilder = new StringBuilder();
+    SecureRandom random = new SecureRandom();
+    IntStream.range(0, 32)
+        .forEach(
+            x -> stringBuilder.append(sampleSpace.charAt(random.nextInt(sampleSpace.length()))));
+
+    return Base64.getEncoder()
+        .encodeToString(stringBuilder.toString().getBytes(StandardCharsets.US_ASCII));
+  }
+
+  public static Status getInstanceStatus(String instanceName) throws IOException {
+    try (InstancesClient instancesClient = InstancesClient.create()) {
+      Instance response = instancesClient.get(PROJECT_ID, ZONE, instanceName);
+      return response.getStatus();
+    }
+  }
+
   @Before
   public void beforeEach() {
     stdOut = new ByteArrayOutputStream();
```

Comment:
If you are going to take it over (I know this happened before this PR), you should save and [restore the printstream](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#restore-system-resources) so other tests can work.

Also, we have

Reply:
Got it Les. Thank you. 👍